src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -220,7 +236,7 @@ public static MetricName metricName(String name, String type, String group) {
         final String sanitisedGroup = sanitise(group);
         final String sanitisedType = sanitise(type);
         final String sanitisedName = sanitise(name);
-        String mBeanName = String.format("%s:type=%s,name=%s", sanitisedGroup, sanitisedType, sanitisedName);
+        String mBeanName = java.lang.String.format("%s:type=%s,name=%s", sanitisedGroup, sanitisedType, sanitisedName);
```

Comment:
Do we need to fullly qualify `String`?

Reply:
Nope, not at all sure why that changed.

Reply:
Resolved in 82bf79507c7c1e76c40808e722e04847cb265ec6